README.md

```diff
@@ -0,0 +1,716 @@
+# UCAN Delegation Specification v1.0.0-rc.1
+
+## Editors
+
+- [Brooklyn Zelenka], [Fission]
+
+## Authors
+
+- [Brooklyn Zelenka], [Fission]
+- [Daniel Holmgren], [Bluesky]
+- [Irakli Gozalishvili], [Protocol Labs]
+- [Philipp Krüger], [Fission]
+
+## Dependencies
+
+- [DID]
+- [IPLD]
+- [UCAN]
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119].
+
+# 0. Abstract
+
+This specification describes the semantics and serialization format for [UCAN] delegation between principals. UCAN Delegation provides a cryptographically verifiable container, batched capabilities, hierarchical authority, and extensible conditions.
+  
+# 1. Introduction
+
+UCAN Delegation is a certificate capability system with runtime-extensibility, ad hoc conditions, cacheability, and focused on ease of use and interoperability. Delegations act as a proofs for [UCAN Invocation]s.
+
+Delegation provides a way to "transfer authority without transferring cryptographic keys". As an authorization system, it is more interested in "what" can be done than a list of "who can do what". For more on how Delegation fits into UCAN, please refer to the [high level spec][UCAN].
```

Comment:
```suggestion
Delegation provides a way to "transfer authority without transferring cryptographic keys". As an authorization system, it is more interested in "what can be done?" than a list of "who can do what?". For more on how Delegation fits into UCAN, please refer to the [high level spec][UCAN].
```

editorial/consistency suggestion but no worries if you dont like it

Reply:
Thanks @gobengo ! Had some issues with the automated merge, so added manually :)